app/controllers/sign_up/completions_controller.rb

```diff
@@ -18,6 +18,7 @@ def show
     def update
       track_completion_event('agency-page')
       update_verified_attributes
+      send_in_person_completion_survey
```

Comment:
@NavaTim starting a thread here arbitrarily regarding the content. I noticed a few small differences between the Figma and the screenshots in this PR. Bolding what we'd like it to read below: 

Fill out a short, anonymous survey and we’ll use your input to help us **get better** at serving your needs.

Your answers to this survey will be protected according **to the following privacy and security standards**.

Reply:
I took the text here from the Spanish and French translation documents rather than the Figma, so that accounts for the difference here. I can update this to match the Figma, though.

Reply:
That would be great, thanks! I'll update the doc you were referencing as well so it's all consistent.

Reply:
Updated English translation with the corrections provided.